src/AppBundle/Repository/OrderRepositoryInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use Sylius\Component\Order\Repository\OrderRepositoryInterface as BaseOrderRepositoryInterface;
+
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+interface OrderRepositoryInterface extends BaseOrderRepositoryInterface
+{
+    public function count();
```

Comment:
this count method is required here? because it is not implemented in the OrderRepository